src/metabase/query_processor.clj

```diff
@@ -40,14 +40,13 @@
 ;;; +----------------------------------------------------------------------------------------------------+
 
 
-(defn structured-query?
-  "Predicate function which returns `true` if the given query represents a structured style query, `false` otherwise."
+(defn mbql-query?
```

Comment:
I disagree. A raw SQL query is not written in MBQL. Otherwise I need to re-write the entire language reference.